test/functional/rpc_rawtransaction.py

```diff
@@ -171,6 +174,67 @@ def getrawtransaction_tests(self):
         block = self.nodes[0].getblock(self.nodes[0].getblockhash(0))
         assert_raises_rpc_error(-5, "The genesis block coinbase is not considered an ordinary transaction", self.nodes[0].getrawtransaction, block['merkleroot'])
 
+    def getrawtransaction_verbosity_tests(self):
```

Comment:
I added a separate test to validate output fields, only that they exists, not their values. 
Is this worth having?